Libs/MRML/Widgets/Resources/UI/qMRMLClipNodeWidget.ui

```diff
@@ -190,6 +190,16 @@
      </property>
     </widget>
    </item>
+   <item row="4" column="0">
+    <widget class="QCheckBox" name="SmoothClippingCheckBox">
```

Comment:
You may keep a checkbox on the GUI to switch between straight cut and whole cell (no need to directly expose all display node parameters on the GUI), but at least revise the name of the checkbox: I think "straight cut" and "whole cell" could be clear enough.